force-app/main/default/aura/customRecommendations/customRecommendations.cmp

```diff
@@ -0,0 +1,42 @@
+<aura:component implements='forceCommunity:availableForAllPageTypes' access="global" controller="userProfileHelper">
+
+   <aura:attribute name="name"
+                   type="String"
+                   access="global" />
+
+   <aura:attribute name="searchHub"
+                   type="String"
+                   access="global" />
+
+   <aura:attribute name="title"
+                   type="String"
+                   access="global" />
+
+   <aura:attribute name="debug"
+                   type="Boolean"
+                   default="false"
+                   access="global" />
+
+   <aura:attribute name="recommendationsContext"
+                   type="String"
+                   access="global" />
+
+   <aura:handler name="init" value="{!this}" action="{!c.doInit}" />
+
+   <div>
+       <article>
+           <header>
+                <h2>
+                    <span>
+                            {!v.title}
+                    </span>
+                </h2>
+           </header>
+            <CoveoV2:SearchUI name="customRecommendations"
```

Comment:
You should use `name="{!v.name}"` here and add `customRecommendations` as the default value for the name attribute.

Reply:
Oups thank you
Actually I think I should have removed the attribute instead, I don't really plan on making many visualforce pages available to the user. `customRecommendations` will be the only one.
I'll fix in next commit